pygac/klm_reader.py

```diff
@@ -734,14 +734,20 @@ def _validate_header(cls, header):
             raise ReaderError('Improper platform id "%s"!' % platform_id)
 
     def get_telemetry(self):
-        """Get the telemetry.
+        """Get the telemetry. Modified by J.Mittaz / University of Reading
```

Comment:
Git blame can provide information of authorship and changes, so please remove the attributions and description of updates from the code.

Also: docstring should have just one summary line, ending with a dot.